### ISSUE TITLE
Output is out of range

### ISSUE THREAD

**Issue by jianzhuwang** [jorge-pessoa/pytorch-msssim#12]
Sorry to bother you. But I used ssim as a loss function, and found some strange outputs. The gray values of the inputs are normalized to [0,1], but the output can be out of this range, like (-100,...), which is meaningless. In other words, the network is optimized toward a wrong direction. I do not know what is the reason, could you please give me some suggestions? btw, the network can work with other loss functions.

**Comment by jorge-pessoa** [comment]
Hello, 

I apologize for the delay in answering. Are you optimizing your network using the SSIM or MS-SSIM metric? If you are still facing this issue I would suggest trying to optimize the network with the most recent changes pushed to this repository which may help (using normalize="relu").

 In the case your problem persists, I would ask you to submit a minimally reproducible example of your problem (if you are able to) or at least some more details on your experiment. Note that the MS-SSIM metric is not particularly stable for training and may cause problems in certain architectures compared to simpler metrics.

Thank you,

**Comment by jorge-pessoa** [comment]
Due to the staleness, I will close this issue. Feel free to reopen a new one if you are still facing the same problem.
